microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -550,11 +548,11 @@ See also:
 <!-- ====================================================================== -->
 ###### PermissionKind.WindowManagement
 
-<!-- todo: merge/coordinate this section w/ the above -->
+<!-- 112 todo: merge/coordinate this section w/ the above -->
```

Comment:
todo: merge/coordinate this section w/ the above, in this 112 PR.  

@novac42 - probably trivial, probably need to just move the below links into the above tabs and get rid of this h2 section.

Reply:
via v111 PR, we ended up only linking to the enum, removed this section / link